README.md

```diff
@@ -34,6 +34,49 @@ This library requires:
 
 This package provides a Cinema 4D plugin that creates jobs for AWS Deadline Cloud using the [AWS Deadline Cloud client library][deadline-cloud-client]. Based on the loaded scene it determines the files required, allows the user to specify render options, and builds an [OpenJD template][openjd] that defines the workflow.
 
+### Submitter Python environment 
+The submitter requires the following modules to be installed:
+
+```sh
+pip install deadline-cloud-for-cinema-4d
+pip install deadline[gui]
+```
+
+### Cinema 4D Plugin Extension Setup 
+
+The Cinema 4D plugin extension can be downloaded from the git repo:
+
+https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/blob/mainline/deadline_cloud_extension/DeadlineCloud.pyp
+
+Place this in a location accessible by C4D instances.
+
+Add the `g_additionalModulePath` environment variable to point to the above location, so that Cinema 4D can load the plugin.
+
+By default, Cinema4D does not see **PYTHON_PATH** and as a result **deadline-cloud-for-cinema-4d** - the extension is programmed to look for an environment variable called `DEADLINE_CLOUD_PYTHONPATH`. Setup this variable to point to the location of your **Python > Lib > site-packages**  - e.g.
+
+```
+C:\Program Files\Python311\Lib\site-packages
+```
+
+If you load a scene, click on **Extensions > Deadline Cloud Submitter** to view the submitter.
+
+### Additional Python Libraries
+
+Some specific versions of Cinema 4d ( e.g. `Cinema 4D 2024.1.0`) have been found to be missing some libraries key to Deadline requirements ; in later versions such as `2024.4.0` this has been resolved. 
+
+A missing library error will manifest in errors that can be visible from the **Python** section of the **Extensions > Console** UI. These typically look like:
+
+```
+PySide6/__init__.py: Unable to import Shiboken from  ...
+```
+
+To remedy these errors, you can switch to a later version of Cinema 4D which resolves the missing libraries, or you can manually add them specifically to the Cinema 4D python module, e.g.
+
+```
+& "C:\Program Files\Maxon Cinema 4D 2024\resource\modules\python\libs\win64\python.exe"-m ensurepip   
```

Comment:
I personally used these commands and I find these to be incredibly useful. Thanks for writing these! 

But I wonder if this should be in `DEVELOPMENT.md` instead or if we should just have all the instructions from that page over here? 
https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/blob/mainline/DEVELOPMENT.md

For example, in blender (https://github.com/aws-deadline/deadline-cloud-for-blender/blob/mainline/DEVELOPMENT.md) and other repos we have installation instructions for submitters in `DEVELOPMENT.md`.

Reply:
Sure - its a little detailed to be on the main page.

Can we add a line in 'Getting Started' to say something like 'For more detailed installation of the submitter and adaptor, please see the development.md' ....

Reply:
Yup that makes sense to me.